Mathlib/FieldTheory/Normal/Basic.lean

```diff
@@ -283,8 +283,8 @@ end minpoly
 /--
 A quadratic extension is normal.
 -/
-instance IsQuadraticExtension.Normal (F K : Type*) [Field F] [Field K]
-    [h : IsQuadraticExtension F K] :
+instance IsQuadraticAlgebra.Normal (F K : Type*) [Field F] [Field K] [Algebra F K]
+    [h : IsQuadraticAlgebra F K] :
```

Comment:
again, if it's a class then the lemmas should be written in such a way that the instance doesn't need to be explicitly named.